public/static/js/spawned_hosts.js

```diff
@@ -326,6 +326,8 @@ mciModule.controller('SpawnedHostsCtrl', ['$scope','$window', '$timeout', 'mciSp
       $scope.curHostData.isTerminated = host.isTerminated;
       // check if this is a windows host
       $scope.curHostData.isWinHost = false;
+      // XXX: if this is-windows check is updated, make sure to also update
+      // model/distro/distro.go as well
       if (host.distro.arch.indexOf('win') != -1) {
```

Comment:
this is starts with, and the server uses contains

Reply:
>The indexOf() method returns the index within the calling String object of the first occurrence of the specified value, starting the search at fromIndex. Returns -1 if the value is not found.

from: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/indexOf

By saying `!= -1`, it's checking for presence. (if it were `== 0`, it would be a prefix check)